### ISSUE TITLE
_source field disappeared from elastic index mapping

### ISSUE THREAD

**Issue by aroscani** [strapdata/elassandra#226]
Hi,
My installed single node Elassandra instance (detail below)

"version": {
        "number": "6.2.3",
        "build_hash": "4011e74",
        "build_date": "2018-06-25T13:15:17.796844Z",
        "build_snapshot": false,
        "lucene_version": "7.2.1",
        "minimum_wire_compatibility_version": "5.6.0",
        "minimum_index_compatibility_version": "5.0.0"
    }

is facing a strange issue that I'm going to explain in detail:

I've created an index starting from Cassandra table with the base command

{
  "settings": {
    "keyspace": "KEYSPACE"
  },
  "mappings": {
    "MAPPING": { "discover": ".*" }
  }
}

then I've populated index with some insert into Cassandra table.
At first, when I submit a _search query with REST client I can see all the data in a correct way (_index, _type, _id, _source, _score).

After some time the index stops to serve _source field so the result looks like this:

 "hits": {
        "total": 33,
        "max_score": 1,
        "hits": [
            {
                "_index": "INDEX",
                "_type": "MAPPING",
                "_id": "[6651,\"ddvstatus\",\"6224-ddvstatus\",1537482660140]",
                "_score": 1
            },

**The _source field disappeared** :(

Maybe I miss some configuration or mistake something else?

PS. I can't see any error or warning related into the log file

**Comment by DBarthe** [comment]
Hello,

You can increase the logging level with `nodetool setlogginglevel <package> TRACE` for packages such as `org.elassandra` and `org.elasticsearch`

If you can provide clear steps that reproduce the issue I would investigate

**Comment by vroyer** [comment]
Hi,
This may happen when cassandra hinted handoff are replay out of time order. Please update to the last release 5.5.0.24 or 6.2.3.7 where this is fixed.
Thanks in advance for your feed back if such issue happen again.

**Comment by Gappa88** [comment]
I have a similar problem.
I have installed Elassandra 6.2.3.7, created a Keyspace and Table in Cassandra and then created the respective index in ES via **PUT API**. 

after inserted some data into Cassandra, using NodeJS cassandra-driver, I noticed that into the ES index there are too many documents respects Cassandra's. This is caused  by some documents that in ES are in some way empty, the only fields present are `_id`, `_index`, `_type` and `_score`

```
....
"hits": {
    "total": 4,
    "max_score": 10.738641,
    "hits": [
      {
        "_index": "myindex",
        "_type": "mytype",
        "_id": """["p1",1543540357000]""",
        "_score": 10.738641
      },
...
```

I have also queried Cassandra for that `_id` and the result is correct, there are no strange entries in there.

What can cause this behaviour?
